pollination/honeybee_radiance/post_process.py

```diff
@@ -311,6 +311,49 @@ def calculate_annual_metrics(self):
     )
 
 
+@dataclass
+class AnnualDaylightEN17037Metrics(Function):
+    """Calculate annual daylight EN 173037 metrics for annual daylight simulation."""
+
+    folder = Inputs.folder(
+        description='This folder is an output folder of annual daylight recipe. Folder '
+        'should include grids_info.json and sun-up-hours.txt. The command uses the list '
+        'in grids_info.json to find the result files for each sensor grid.',
+        path='raw_results'
+    )
+
+    schedule = Inputs.file(
+        description='Path to an annual schedule file. Values should be 0-1 separated '
+        'by new line. This should be a daylight hours schedule according to EN 17037.',
+        path='schedule.txt'
+    )
+
+    @command
+    def calculate_annual_metrics_en17037(self):
+        return 'honeybee-radiance post-process annual-daylight-en17037 raw_results ' \
+            'schedule.txt --sub_folder ../metrics'
```

Comment:
We should keep the output folder in the same layer in Function. You can always copy it to a top level folder in the recipe.

```suggestion
            'schedule.txt --sub_folder metrics'
```